### DISCUSSION TITLE
tui-input-date-range class="b-form" (large picker) does not work

### DISCUSSION THREAD

**Discussion by sommmen** [taiga-family/taiga-ui#5019] [Q&A] [answered]
Hiya,

I went through the normal installation process to be able to use the date range picker, but for some reason i can't get the large picker to show up.

https://taiga-ui.dev/components/input-date-range#large

![image](https://github.com/Tinkoff/taiga-ui/assets/8641495/f5f1c128-2da9-4ca2-87b9-c9541bc0eba9)

The picker shows up fine and works great - so i don't understand why this would not work.
Any clues on why this would happen?

```
<tui-input-date-range
    class="b-form"
    [formControl]="control"
    [items]="shortcuts"
>
Kies een datum
</tui-input-date-range>

...

  "@taiga-ui/cdk": "^3.37.0",
  "@taiga-ui/core": "^3.37.0",
  "@taiga-ui/icons": "^3.37.0",
  "@taiga-ui/kit": "^3.37.0",    
...

  "src/web.config",
  {
    "glob": "**/*",
    "input": "node_modules/@taiga-ui/icons/src",
    "output": "assets/taiga-ui/icons"
  }
],
"styles": [
  "node_modules/@taiga-ui/core/styles/taiga-ui-theme.less",
  "node_modules/@taiga-ui/core/styles/taiga-ui-fonts.less",
  "./node_modules/bootstrap/dist/css/bootstrap.min.css",
  "src/styles.css"
],
```

**Comment by waterplea** [comment, marked as answer]
If you pass `items` it shows a single month and a list of items on the side. Without items it shows two adjacent months.
